packages/common/src/TransactionUtils.ts

```diff
@@ -76,6 +86,16 @@ export const runTransaction = async ({
   // This method does not play nicely in tests. Leave the nonce null to auto fill.
   else if (argv.network != "test") transactionConfig.nonce = await web3.eth.getTransactionCount(transactionConfig.from);
 
+  // Store the transaction nonce in the web3 object so that it can be used in the future. This enables us to fire a
+  // bunch of transactions off without needing to wait for them to be included in the mempool by manually incrementing.
+  if ((web3 as AugmentedWeb3).nonces?.[transactionConfig.from])
```

Comment:
The stored nonces will reset on each serverless run right? So is it possible that you'll get nonce collisions if the transactions don't mine fast enough between serverless runs?

Reply:
that's correct